addons/website/views/website_templates.xml

```diff
@@ -255,6 +255,12 @@
     </xpath>
 </template>
 
+<template id="header_logo_none" inherit_id="website.layout" name="No Logo" active="False">
+    <xpath expr="//header//a[hasclass('navbar-brand')]" position="replace">
+        <a href="/" class="navbar-brand d-none"/>
```

Comment:
Can't this be just removed?

Reply:
If we remove it we will have issues where we xpath it. When we switch header templates for example

Reply:
> Same remark for every option like that. Also, instead of an option which hides the header, would this be possible to extract the header in a template to have a xml template which adds the header. That way, any xpath targeting the header should inherit from this template and it could be safely disabled.

But even with removal, it would just be a matter of setting a high sequence on the removal template.

Reply:
Do we really need to remove it ? As it's a page option I dont find it's bad if it's stay in the DOM...
Actually if I remove it I have some issues with header templates (hamburger menu)

Reply:
if we remove it we have issues with hamburger header templates. So as discussed we leave it like that ...